src/compiler/compile/render_dom/wrappers/InlineComponent/index.ts

```diff
@@ -23,6 +23,8 @@ import compiler_warnings from '../../../compiler_warnings';
 
 type SlotDefinition = { block: Block; scope: TemplateScope; get_context?: Node; get_changes?: Node };
 
+const regex_invalid_variable_identifier_characters = /[^a-zA-Z_$]/g;
```

Comment:
there are some similar regex patterns
https://github.com/ivanhofer/svelte/blob/more-performant-regex/src/compiler/compile/render_dom/wrappers/Element/index.ts#L144
https://github.com/ivanhofer/svelte/blob/more-performant-regex/src/compiler/compile/render_dom/wrappers/shared/add_actions.ts#L15

But this one is missing the `0-9` part. Should it be added?

Reply:
doesnt really matter.

i wonder if we should extract the common regexes into a separate file.

seems like a lot of regex are kind of similar

Reply:
I was thinking the same.
There are already defined some regexes here: https://github.com/sveltejs/svelte/blob/more-performant-regex/src/compiler/utils/patterns.ts

I could move common regexes and maybe rename the existing ones so they share the same naming pattern?